seqr/migrations/0046_auto_20180803_1708.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2018-08-03 17:08
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('seqr', '0045_auto_20180801_1933'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AnalysisGroup',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('guid', models.CharField(db_index=True, max_length=30, unique=True)),
+                ('created_date', models.DateTimeField(db_index=True, default=django.utils.timezone.now)),
+                ('last_modified_date', models.DateTimeField(blank=True, db_index=True, null=True)),
+                ('name', models.TextField()),
+                ('description', models.TextField(blank=True, null=True)),
+                ('created_by', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='+', to=settings.AUTH_USER_MODEL)),
+                ('families', models.ManyToManyField(to='seqr.Family')),
+                ('project', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='seqr.Project')),
+            ],
+        ),
+        migrations.AlterUniqueTogether(
+            name='analysisgroup',
+            unique_together=set([('project', 'name')]),
```

Comment:
because it would be incredibly confusing to have a list of analysis groups shown on the project page where 2 have the same name and theres no real way for the user to know the difference between them, and theres no good reason to have duplicated names. I can add better error handling for that though